src/js/facility-locator/reducers/searchQuery.js

```diff
@@ -0,0 +1,22 @@
+import { SEARCH_QUERY_UPDATED } from '../actions';
+// TODO (bshyong): flesh out shape of service Type object/options
+const INITIAL_STATE = {
+  searchString: '',
+  serviceType: null,
+  position: {
```

Comment:
We should decide what our default location should be if someone decides to block geolocation.